snippets/csharp/System/NullReferenceException/Overview/nullreturn2a.cs

```diff
@@ -1,40 +1,43 @@
-﻿// <Snippet5>
-using System;
+﻿using System;
 
-public class Example
+namespace NullReturn2
 {
-   public static void Main()
-   {
-      Person[] persons = Person.AddRange( new String[] { "Abigail", "Abra",
+    // <Snippet5>
+    public class NullReturn2Example
+    {
+        public static void Main()
+        {
+            Person[] persons = Person.AddRange([ "Abigail", "Abra",
                                           "Abraham", "Adrian", "Ariella",
-                                          "Arnold", "Aston", "Astor" } );
-      String nameToFind = "Robert";
-      Person found = Array.Find(persons, p => p.FirstName == nameToFind);
-      if (found != null)
-         Console.WriteLine(found.FirstName);
-      else
-         Console.WriteLine("{0} not found.", nameToFind);
-   }
-}
+                                          "Arnold", "Aston", "Astor" ]);
+            string nameToFind = "Robert";
+            Person found = Array.Find(persons, p => p.FirstName == nameToFind);
+            if (found != null)
+                Console.WriteLine(found.FirstName);
+            else
+                Console.WriteLine("{0} not found.", nameToFind);
```

Comment:
string interpolation:

```suggestion
                Console.WriteLine($"{nameToFind} not found.");
```